src/dali_executor/dali_pipeline.h

```diff
@@ -150,6 +150,13 @@ class DaliPipeline {
     return num_threads_;
   }
 
+  static void LoadPluginLibs(const std::vector<std::string>& plugin_paths) {
+    InitDali();
+    for (auto&& path : plugin_paths) {
```

Comment:
Don't create an illusion of being able to move these:
```suggestion
    for (const auto& path : plugin_paths) {
```